Sources/BraintreeCard/CreditCardGraphQLBody.swift

```diff
@@ -0,0 +1,186 @@
+// swiftlint:disable nesting
```

Comment:
Can we move this down to above our docstring on the struct to match other files?